src/application/ApplicationDetail/SenderAPI.tsx

```diff
@@ -40,7 +41,20 @@ export class SenderAPI extends Component<Props, State> {
     };
   }
 
+  async componentDidMount() {
+    this.setState({ docLinks: await Config.getInstance().getDocsConfig() });
+  }
+
```

Comment:
This code and getLinks() below is duplicated a lot. Should the links be something that should be at the context level?